buildSrc/src/main/java/org/elasticsearch/gradle/ReaperService.java

```diff
@@ -24,24 +28,20 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-public class ReaperService {
+public abstract class ReaperService implements BuildService<ReaperService.Params>, AutoCloseable {
+
+    interface Params extends BuildServiceParameters {
```

Comment:
nit: can we throw this at the bottom of the class as it typical with other inner-classes?